src/source/PeerConnection/DataChannel.h

```diff
@@ -14,8 +14,8 @@ typedef struct {
     RtcDataChannel dataChannel;
 
     PRtcPeerConnection pRtcPeerConnection;
+    PRtcDataChannelInit pRtcDataChannelInit;
```

Comment:
Danger! In this case, you will be storing the pointer to an object that was passed in by the customer. This implies that the object has to survive throughout the duration of the lifetime of your object. This might not be the case when it comes to the customers application. We also specifically do not require it.

Suggestion: Instead of a pointer, have a full structure here. Copy the structure in the createDataChannel API and use the copy whenever needed.

Reply:
Oops. You are right. Fixed.